airflow/serialization/serialized_objects.py

```diff
@@ -407,6 +407,8 @@ def serialize_to_json(
                 serialized_object[key] = encode_timetable(value)
             elif key == "dataset_triggers":
                 serialized_object[key] = cls.serialize(value)
+            elif key == "dataset_triggers":
+                serialized_object[key] = cls.serialize(value)
```

Comment:
```suggestion
            elif key == "dataset_triggers":
                serialized_object[key] = cls.serialize(value)
```

Reply:
Changed it. I think it was messed during rebased